breaking-front/src/mocks/postEditHandlers.js

```diff
@@ -0,0 +1,9 @@
+import { API_PATH } from 'constants/path';
+import { rest } from 'msw';
+
+export const postEditHandlers = [
+  rest.put(API_PATH.POST_EDIT('*'), (req, res, ctx) => {
```

Comment:
```suggestion
  rest.put(API_PATH.POST_EDIT(':postId'), (req, res, ctx) => {
```
* 표시는 querystring일 때만 쓰면 좋겠습니다!